icons/toilet-roll.svg

```diff
@@ -0,0 +1,17 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <ellipse cx="10" cy="8" rx="3" ry="2" />
```

Comment:
What about making the center circle smaller?
![icon](https://github.com/lucide-icons/lucide/assets/25524993/4ad23470-fd8a-4896-9ff3-77f29a51fb90)

```suggestion
  <ellipse cx="10" cy="8" rx="2.5" ry="2" />
```

Reply:
> What about making the center circle smaller?

That would put it off grid and make it blurry at 1x.

Reply:
> That would put it off grid and make it blurry at 1x.

We need to implement this https://github.com/jguddas/lucide/commit/fa7623c2f433230cecd5ef740784e08b2eff3bc7, also seems like something has gone wrong with the existing preview bot! 😕